packages/editor/src/browser/editor-command.ts

```diff
@@ -57,6 +60,9 @@ export namespace EditorCommands {
 @injectable()
 export class EditorCommandContribution implements CommandContribution {
 
+    @inject(PreferenceService)
+    protected readonly preferencesService: PreferenceService;
```

Comment:
please use statically typed preference interfaces, i.e. `EditorPreferences`

with them constants for properties are redundant, tsc does not allow to use any other properties

Reply:
I'm not sure that I understood you correctly, Is it related to this: https://github.com/vinokurig/theia/blob/f2d71426676430d19d2cf3498b78e135a291c8c6/packages/editor/src/browser/editor-command.ts#L61 ?